frontend/src/global_styles/layout/work_packages/_full_view.sass

```diff
@@ -79,7 +79,7 @@
       display: grid
       grid-template-rows: auto auto 1fr
       height: 100%
-      padding: 5px 0 10px 15px
+      padding: 5px 0 0px 15px
```

Comment:
This will change padding on other tabs as well

Reply:
Fixed